### ISSUE TITLE
In GH PR review page, headers have redundant url content

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode-pull-request-github#7509]
On github.com, the header content doesn't have these URLs as a prefix

<img width="347" height="273" alt="Image" src="https://github.com/user-attachments/assets/e792f9e6-f0c1-4466-9358-b2a3b36a524d" />

**Comment by meganrogge** [comment]
cc @jooyoungseo

**Comment by meganrogge** [comment]
@jooyoungseo, could you please let us know if this is fixed? I used `ctrl+option+u` on macOS with VoiceOver and was presented with a very long list of links. I did not see that extra content in there, so considering this verified.

**Comment by jooyoungseo** [comment]
@meganrogge Yes, it got fixed! Thank you very much!
